spec/factories/user.rb

```diff
@@ -1,4 +1,8 @@
 FactoryGirl.define do
   factory :user do
+    name     "Some User"
```

Comment:
Put one space between the method name and the first argument.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.